Paradigm4/revealgenomics-apps
# Changes to match rest api test

Multiple changes to pass REST API test (python script). Entire list is being maintained at #29. This PR makes sure that the following 4 tests pass:

- Get list of all projects (`test1_GET_projects`)
- Create a new project (`test2_POST_project`)
- Retrieve the newly created project (`test3_GET_project`)
- Update the project (`test4_PUT_project`)

Some of the major changes include
- use `unboxedJSON` serializer instead of default 
- format result to return JSON dictionary instead of JSON list
- rename `projectId->projectid, project_name->projectid, sampleId->sampleid, sample_name->sampleid`
- Successful `POST` should return `201 (Created)` instead of 200
- `GET /project` should not return DB specific fields
